BESMod/Systems/Demand/Building/TEASERThermalZone.mo

```diff
@@ -2,9 +2,9 @@
 model TEASERThermalZone
   "Reduced order building model, coupled with TEASER"
   extends BaseClasses.PartialDemand(
-    ARoo=sum(zoneParam.ARoof)/nZones,
-    hBui=sum(zoneParam.VAir)^(1/3),
-    ABui=2*sum(zoneParam.VAir)^(1/3),
+    hBui=0,
+    ABui=0,
+    ARoo=0,
```

Comment:
Alright, now we only need to check if a redeclaration is required in the other examples. Actually, I am not sure, we even should set a default. Hence, the user is forced to set the Parameters.

Reply:
It's not, CI passed with success. For now, 0 is better than no default, as it's only required in some cases.